index.bs

```diff
@@ -2011,6 +2018,31 @@ remove optional flag from the field.
 
 </div>
 
+<div algorithm>
+To <dfn>serialize an Array-like</a> given |production|, |value|, |handle id|,
```

Comment:
```suggestion
To <dfn>serialize an Array-like</dfn> given |production|, |value|, |handle id|,
```

Reply:
That issue was fixed by the last update.